src/Client/components/submission/code-editor.tsx

```
@@ -154,32 +146,13 @@
             <Card className="border-none rounded-lg p-4 shadow-none bg-white dark:bg-[#18181b]">
                 <h2 className="text-xl font-bold mb-3">Problem Statement</h2>
                 <div className="space-y-4">
+                    <p dangerouslySetInnerHTML={{ __html: question.statementMarkdown }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)